.github/workflows/schemacode_ci.yml

```diff
@@ -5,92 +5,81 @@ on:
     branches:
       - "master"
       - "autodata2"
-    paths:
-      - "tools/schemacode/**"
-      - "src/schema/**"
     tags:
       - "schema-*"
   pull_request:
     branches:
       - "*"
-    paths:
-      - "tools/schemacode/**"
-      - "src/schema/**"
 
-jobs:
+defaults:
+  run:
+    shell: bash
 
-  run_tests:
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        python-version: ["3.10"]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: "Install build dependencies"
+        run: pip install --upgrade build twine
+      - name: "Install test dependencies on tag"
+        run: pip install --upgrade pytest pyyaml pandas tabulate
+        if: ${{ startsWith(github.ref, 'refs/tags/schema-') }}
+      - name: "Build archive on tag"
+        run: pytest tools/schemacode/bidsschematools -k make_archive
+        env:
+          BIDSSCHEMATOOLS_RELEASE: 1
+        if: ${{ startsWith(github.ref, 'refs/tags/schema-') }}
```

Comment:
These are the only differences between `schema-*` tag builds and otherwise. The idea is that we should be able to use the same process regardless.

Reply:
Sounds good. Did you test this?

Reply:
Yes.